changelog.md

```diff
@@ -1,3 +1,13 @@
+### UNRELEASED
+### **Bravas**
+---
+* LLT-4970: PQ VPN is enabled by calling `telio_connect_to_exit_node_postquantum()`
```

Comment:
Can you remove this entry from the changelog for the previous release? It must have been a mistake on my end

Reply:
Yeh I moved it from "v4.3.0-rc0"